### PR TITLE
add 4 pull-requests: community standards, donate, deploy with ci/cd and website for docs, faq or tutorials

### PR DESCRIPTION
**Pull request by ghost**
Hello everyone!

## Intro
I would like to contribute to this pull-request and I hope you all accept it. So, things I added:
- CODE_OF_CONDUCT.md
- CONTRIBUTING.md
- Issue/pull-request template
- deploy with ci/cd bere: appveyor.yml
- donate here:  FUNDING.yml
- github-pages for docs with static.yml

*demo here: img-1*
image

*demo here: img-2*
image

## pull-request-1
I added files in markdown that make up a community standards area.  You can see the before and after. These files serve to guide people who want to develop things for Foton. This pull-request-1 composes: CODE_OF_CONDUCT.md CONTRIBUTING.md, Issue/pull-request template in folder called: `.github`.

*img-before*
image

*img-after*
image

## pull-request-2

Here I added a responsible file to add financial support in the open project: foton. You can see the before and after. This pull-request-2 composes: FUNDING.yml in folder called: `.github`.

*img-before*
image

*img-after*
image

## pull-request-3
Now it will be possible to automate all tests and builds of the library in C# Foton. You can test the library on different operating systems in the cloud with AppVeyor. AppVeyor has a free plan for open projects. AppVeyor is Continuous Integration(CI) and Deployment (CD) service.

## pull-request-4
I added in the pull-request here a static file to generate a website with docsify, github pages, markdown for docs. Please, see this: [https://cliffbellman-dev.github.io/foton/#/](https://cliffbellman-dev.github.io/foton/#/).  What I'm sending here might be something like: `https://integrativesoft.github.io/foton/#/`.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+
+<br/>
+<div align="center">
+      <img src="https://img.shields.io/badge/c%23-%23239120.svg" alt="Csharp CI/CD">
+  </a>
+  <a href="https://opensource.org/licenses/Apache-2.0">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="Foton is licensed with Apache License 2.0">
+  </a>
+  <a href="https://www.repostatus.org/#active">
+  <img src="https://www.repostatus.org/badges/latest/active.svg" alt="Project Status: Active – The project has reached a stable, usable state and is being actively developed."/>
```
Comment (pablocar80) [inline docs/README.md:24]: I wouldn’t say it’s stable/usable yet, still experimental

File: docs/README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+
+</div>
+
+<br/>
+
+## About
+Foton is prototype for an Electron alternative written in C# by Integrative Software LLC and contributors.
+
+## Quick start
+First, download and install C# in Visual Studio - last version is required.
```
Comment (pablocar80) [inline docs/README.md:36]: VSCode should be an option too

**Review by pablocar80 — CHANGES_REQUESTED** [review]
Please check comments, thank you

File: README.md [hunk 3]
```diff
@@ -20,8 +20,8 @@
   <a href="https://opensource.org/licenses/Apache-2.0">
     <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="Foton is licensed with Apache License 2.0">
   </a>
-  <a href="https://www.repostatus.org/#active">
-  <img src="https://www.repostatus.org/badges/latest/active.svg" alt="Project Status: Active – The project has reached a stable, usable state and is being actively developed."/>
+  <a href="https://www.repostatus.org/#experimental">
+  <img src="https://github.com/GIScience/badges/raw/master/status/experimental.svg" alt="Project Status: Experimental – The project has reached a stable, usable state and is being actively developed."/>
```
Comment (pablocar80) [inline README.md:24]: it has not yet reached a stable state (we may need to change the API without backwards compatibility)

File: README.md [hunk 4]
```diff
@@ -33,7 +33,7 @@
 Foton is prototype for an Electron alternative written in C# by Integrative Software LLC and contributors.
 
 ## Quick start
-First, download and install C# in Visual Studio - last version is required. 
+First, download and install C# in Visual Studio or VSCode - last version is required.
```
Comment (pablocar80) [inline README.md:36]: Let's remove this section, it doesn't really say anything

File: docs/README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+<br/>
+<div align="center">
+      <img src="https://img.shields.io/badge/c%23-%23239120.svg" alt="Csharp CI/CD">
+  </a>
+  <a href="https://opensource.org/licenses/Apache-2.0">
+    <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" alt="Foton is licensed with Apache License 2.0">
+  </a>
+  <a href="https://www.repostatus.org/#experimental">
+  <img src="https://github.com/GIScience/badges/raw/master/status/experimental.svg" alt="Project Status: Experimental – The project has reached a stable, usable state and is being actively developed."/>
```
Comment (pablocar80) [inline docs/README.md:24]: remove 'stable' from here too

**Review by pablocar80 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Hello.

Please, I hope this pull-request is good, workable. So, check all.

**Comment by pablocar80** [comment]
approved, feel free to merge

**Review by ghost — APPROVED** [review]
Hello.

For some reason I can't merge. 

*I made a branch for a pull-request, and I expect the pull-request to pass.*

Please, confirm this [add 4 pull-requests: community standards, donate, deploy with ci/cd and website for docs, faq or tutorials](https://github.com/integrativesoft/foton/pull/15)

**Review by ghost — APPROVED** [review]
Hi @pablocar80.

> Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.

Please, add your merge or delete or close this pull-request.

**Comment by ghost** [comment]
Hi @pablocar80, thanks!

**Pull request merged**